### ISSUE TITLE
Dictionary shows only one definition

### ISSUE THREAD

**Issue by enigmasc** [koreader/koreader#6863]
* KOReader version: v2020.10.1-42-g25cfef3_2020-11-09
* Device: Kobo Touch, Kobo Glow HD, & Kobo Clara HD 

#### Issue

Some stardicts' format is like this:
----
Washington	Washington1 /ˈwɒʃɪŋtən/ \n1. a state of the north-western US, on the Pacific coast; pop. 6,549, 224  (est. 2008); capital, Olympia. It became the 42nd state in 1889. \n2. the capital of the US; pop. 591,833  (est. 2008). It is coextensive with the District of Columbia, a federal district on the Potomac River with boundaries on the states of Virginia and Maryland. Founded in 1790, during the presidency of George Washington, it was planned and built as a capital city. Full name WASHINGTON DC. \n3. an industrial town in NE England, designated as a new town in 1964; pop. 51,700  (est. 2009). \n
Washington	Washington2 /ˈwɒʃɪŋtən/ \nWashington2, Booker T.  (1856-1915), American educationist; full name <i>Booker Taliaferro Washington</i>. A leading commentator for black Americans, Washington established the Tuskegee Institute in Alabama  (1881). His support for segregation and his emphasis on black people's vocational skills attracted criticism from other black leaders. \n
Washington	Washington3 /ˈwɒʃɪŋtən/ \nWashington3, George  (1732-99), American general and statesman, 1st President of the US 1789-97. Washington helped win the War of Independence by keeping his army together through the winter at Valley Forge and winning a decisive battle at Yorktown  (1781). He chaired the convention at Philadelphia  (1787) that drew up the American Constitution and subsequently served two terms as President, following a policy of neutrality in international affairs.\n
---

When a word that has 2 or more definitions is selected to find its meanings, Stardict always shows the second one only. In this case, it'd only tell you who Washington, Booker T. is. It is an old problem but has never been dealt with, I believe. 

Can anyone work on it? Thanks.

**Comment by NiLuJe** [comment]
An actual minimal test-case and CLI sdcv repro would be helpful.

**Comment by enigmasc** [comment]
[Stardict error.zip](https://github.com/koreader/koreader/files/5518084/Stardict.error.zip)
![stardict error](https://user-images.githubusercontent.com/74174080/98690101-8c94ba80-23a7-11eb-8208-c0bf734532a5.jpg)

 A sample dictionary is here, and it is how it looks like on Goldendict (Windows ver.)

**Comment by poire-z** [comment]
It might be a sdcv issue, if I trust my comment at https://github.com/koreader/koreader/issues/5437#issuecomment-536499455:
> because sdcv shows only the first one... so you might not even know that your dict has multiple distinct definitions for a same word)

**Comment by NiLuJe** [comment]
Yup.

```
┌─(ROOT@dahlia:pts/0)──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────(~)─┐
└─(0.47:49%:16:07:66%:#)── /mnt/onboard/.adds/koreader/sdcv -2 . "Washington"                                                                                                                                                                                                                                                                          ──(Tue, Nov 10)─┘
save to cache ./stardict error.idx
Found 1 items, similar to Washington.
-->stardict error
-->Washington

Washington1 /ˈwɒʃɪŋtən/ 
1. a state of the north-western US, on the Pacific coast; pop. 6,549, 224  (est. 2008); capital, Olympia. It became the 42nd state in 1889. 
2. the capital of the US; pop. 591,833  (est. 2008). It is coextensive with the District of Columbia, a federal district on the Potomac River with boundaries on the states of Virginia and Maryland. Founded in 1790, during the presidency of George Washington, it was planned and built as a capital city. Full name WASHINGTON DC. 
3. an industrial town in NE England, designated as a new town in 1964; pop. 51,700  (est. 2009).
```

**Comment by NiLuJe** [comment]
c.f., #2951 & #5168

TL;DR: As @poire-z said, the workaround is to "fix" the dictionary, not sdcv ;).
